src/auto/authentication_request.rs

```diff
@@ -18,18 +17,19 @@ use glib::signal::SignalHandlerId;
 use glib::signal::connect_raw;
 use glib::translate::*;
 #[cfg(any(feature = "v2_2", feature = "dox"))]
-use glib_ffi;
+use glib_sys;
 #[cfg(any(feature = "v2_2", feature = "dox"))]
 use std::boxed::Box as Box_;
 use std::fmt;
 #[cfg(any(feature = "v2_2", feature = "dox"))]
 use std::mem::transmute;
+use web_kit2_sys;
```

Comment:
Could you please configure it to generate the name `webkit2_sys` instead?

Reply:
You meant add hardcoded replacement to gir?
Or I totally forgot about way to "configure" it.

Reply:
Added option to gir https://github.com/gtk-rs/gir/pull/745, will update this PR soon